hapi-server.http

```diff
@@ -0,0 +1,16 @@
+POST http://localhost:8080/fhir/DEFAULT/$partition-management-create-partition
```

Comment:
some comments on how to use this file for the uninitiated?

Reply:
Leftover, will remove